server/src/schema.graphql

```diff
@@ -4,6 +4,7 @@ type Query {
   me: User
 
   getCategories: [Category]
+  getJobs(id: ID!): [Job!]!
```

Comment:
would be more clear if argument was projectId

Reply:
fixed